docs/Configuring-Pantheon/Consensus-Mechanisms.md

```diff
@@ -46,9 +44,19 @@ increases number as the of validators increases.
 
 For IBFT 2.0, the time to add new blocks increases as the number of validators increases.   
 
-### Minimum Number of Validators 
+### Type of Byzantine-fault-tolerance
```

Comment:
Maybe this is "Impact of Bad Actors"?
- Ibft tolerates bad actors and fulfils its stated behaviours while bad actors are less than 1/3. After that, forks and re-orgs may occur (from any height after which sufficient nodes have been compromised)? Or can we not re-write history?
- Clique will always exhibit minor forking - however if more than 50%(?) of network is compromised the history of the chain can be re-written.

Reply:
In IBFT if more than 1/3 of actors are Byz, then the entire network is in a state that the network should never be in

Reply:
Name remained at is.  Added points from Trent to existing section.